.github/workflows/merged.yml

```diff
@@ -0,0 +1,44 @@
+name: push main
+
+on:
+  pull_request:
+    branches:
+      - main
+    types: [closed]
+
+jobs:
+  update_version:
+    runs-on: ubuntu-20.04
+    if: github.event.pull_request.merged == true
+    timeout-minutes: 10
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: setup python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+
+      - name: install packages
+        run: >-
+          python -m pip install -r proto/requirements.txt
+
+      - name: protobuf build
+        run: |
+          cd proto
+          make
+
+      - name: update .proto_hash
+        run: |
+          cd proto
+          proto_hash=$(python3 proto_hash.py)
+          if [ ${proto_hash} != $(cat .proto_hash) ]; then
+            echo ${proto_hash} > .proto_hash
+            python3 increment_minor.py $(cat .version) > .version
```

Comment:
This event commits two files:
- .proto_hash
  this file specifies sha256 of `otaclient_pb2.v2.otaclient_pb2.DESCRIPTOR.serialized_pb`
  If the hash is updated, this .proto_hash is updated to this value.
- .version
  this file specifies proto package version.
  If the hash is updated, the minor version of this .version is incremented.
  (Major version is not updated since we need to keep backward compatibility)